tool/src/org/antlr/v4/automata/LexerATNFactory.java

```diff
@@ -412,8 +412,7 @@ else if ("popMode".equals(command) && arg == null) {
 		}
 		else if ("mode".equals(command) && arg != null) {
 			String modeName = arg.getText();
-			checkMode(modeName, arg.token);
-			Integer mode = getConstantValue(modeName, arg.getToken());
+			Integer mode = getModeConstantValue(modeName, arg.getToken());
```

Comment:
1. Ok, I'll revert old error types and numbers.
2. Ok, I'll revert old message format for such errors.